react/Summary.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react'
+import { ExtensionPoint } from 'vtex.render-runtime'
+import { Button } from 'vtex.styleguide'
+import SummaryItem from './components/SummaryItem'
+
+const minTotalizerValue = 0
+
+const Summary: StorefrontFunctionComponent<SummaryProps> = ({
+  subtotal,
+  tax,
+  total,
+  currency,
+}) => {
+  return (
+    <div className="ph5 ph0-l">
+
+      <h5 className="t-heading-5 mt6 mb6 mt8-l pt8-l">Summary</h5>
+      <ExtensionPoint id="coupon" />
+
+      <SummaryItem
+        label="Subtotal"
```

Comment:
This should be a map of the totalizers object not the specific instanciation of the summary item.